powersimdata/data_access/data_access.py

```diff
@@ -193,19 +192,15 @@ def push(self, file_name, checksum):
 class SSHDataAccess(DataAccess):
     """Interface to a remote data store, accessed via SSH."""
 
-    def __init__(self, root=server_setup.DATA_ROOT_DIR):
+    def __init__(self, _fs=None):
         """Constructor"""
-        super().__init__(root)
-        self._fs = None
+        super().__init__()
+        self.root = server_setup.DATA_ROOT_DIR
+        self._fs = _fs
         self.local_fs = fs.open_fs(server_setup.LOCAL_DIR)
 
     @property
     def fs(self):
-        """Get or create the filesystem object
-
-        :raises IOError: if connection failed or still within retry window
-        :return: (*fs.multifs.MultiFS*) -- filesystem instance
-        """
```

Comment:
Wonder why we remove these doc strings?

Reply:
I initially removed the property but put it back later. Looking at it again, the old docstring is not accurate, so will add a new one.